src/modules/tools/tools.scss

```diff
@@ -0,0 +1,32 @@
+@import "~asab-webui/styles/constants/index.scss";
+
+:root[data-theme="light"] {
+  --bg: var(--white);
```

Comment:
@Aringoaway Use tabs not spaces as a padding in the scss file. Also the tab width should be `4`